test/threads.jl

```diff
@@ -449,3 +449,30 @@ function test_nested_loops()
     end
 end
 test_nested_loops()
+
+@testset "libatomic" begin
+    prog = """
+    using Base.Threads
+    function unaligned_setindex!(x::Atomic{UInt128}, v::UInt128)
+        Base.llvmcall(\"\"\"
+            %ptr = inttoptr i$(Sys.WORD_SIZE) %0 to i128*
+            store atomic i128 %1, i128* %ptr release, align 8
+            ret void
+        \"\"\", Void, Tuple{Ptr{UInt128}, UInt128}, unsafe_convert(Ptr{UInt128}, x), v)
+    end
+    code_native(STDOUT, unaligned_setindex!, Tuple{Atomic{UInt128}, UInt128})
+    """
+
+    mktempdir() do dir
+        file = joinpath(dir, "test23901.jl")
+        write(file, prog)
+        run(pipeline(ignorestatus(`$(Base.julia_cmd()) --startup-file=no $file`),
+                     stdout=joinpath(dir, "out.txt"),
+                     stderr=joinpath(dir, "err.txt"),
+                     append=false))
+        out = readchomp(joinpath(dir, "out.txt"))
+        err = readchomp(joinpath(dir, "err.txt"))
+        @test contains(out, "libat_store") || contains(out, "atomic_store")
```

Comment:
That means we couldn't resolve `libatomic` (which we should probably add as a official dependency now).
This test intentionally issues a maligned store so that we don't use hardware features.

Reply:
Do you have https://apps.fedoraproject.org/packages/libatomic installed?

Reply:
OK, I just had to install it to fix the problem. But why isn't it listed in the required dependencies?

Reply:
Just an oversight on my part. Will add it.